app/controllers/VerifiedEmailController.scala

```diff
@@ -53,6 +55,23 @@ class VerifiedEmailController @Inject() (
     }
   }
 
+  def getVerifiedEmailV2(): Action[AnyContent] = authorisedRequest async {
```

Comment:
Could you please remove () as these are redundant? If you face any issues with the routes after removing these (), then leave as is but you should not face any issues